mysite/settings.py

```diff
@@ -103,6 +104,7 @@
 
 
 STATIC_URL = 'static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
```

Comment:
Please, rewrite this to use `pathlib.Path` instead (it's already imported, anyway!)

Reply:
(you may need to wrap that with an `str()` call but I'm not sure)

Reply:
What did you edit exactly and how?

Reply:
Now just locally like `STATIC_ROOT = BASE_DIR / 'static'`. I am writing tests and haven't pushed it yet.

Reply:
Marking this thread as resolved.